src/frontend/src/components/nav/Header.tsx

```diff
@@ -100,12 +100,13 @@ export function Header() {
 
 function NavTabs() {
   const { classes } = InvenTreeStyle();
-  const { tabValue } = useParams();
+  const defaultTabValue = 'home';
+  const tabValue = useLocation().pathname.split('/')[1] || defaultTabValue;
```

Comment:
Not sure that is a great sulution, this is indirectly hardcoded to the current schema - which might change at any point. Thoughts @wolflu05 ?

Reply:
Yes, this would also be my first concern, as we have the feature to configure a relative path where PUI is hosted. Maybe we can use that setting to somehow strip the prefix or find a solution to only get the url from after the prefix. 

And secondly it would be better to wrap the .split and parsing of the panel name into a useMemo hook, so this is in a state and does not cause a rerendering.